scripts/lib/libvirt/fixtures/cloud-admin.xml

```diff
@@ -5,7 +5,7 @@
   <vcpu>1</vcpu>
   <os>
     <type arch='x86_64' machine='pc-0.14'>hvm</type>
-
+    
```

Comment:
This was intentionally done to pass the test. Unfortunately, the tests do a byte-by-byte comparison to test the xml files generated. So the spaces ensure the tests dont throw exception due to space-mismatch. A better fix is to update the tests to compare xml using correct library which can be another PR IMO.